### PR TITLE
[AV-1886] add e2e test for multiple deploy

### PR DESCRIPTION
**Pull request by felipemadero**

### CODE REVIEW

**Review by dasconnor — COMMENTED** [review]
tests look great!

File: tests/e2e/subnet/suite.go [hunk 1]
```diff
@@ -65,10 +65,10 @@ var _ = ginkgo.Describe("[Subnet]", func() {
 
 		deployOutput = commands.DeploySubnetLocally(subnetName)
 		rpcs, err = utils.ParseRPCsFromOutput(deployOutput)
-		if err != nil {
+		if err == nil {
 			fmt.Println(deployOutput)
 		}
-		gomega.Expect(err).Should(gomega.BeNil())
+		gomega.Expect(err).Should(gomega.HaveOccurred())
```
Comment (dasconnor) [inline tests/e2e/subnet/suite.go:71]: I don't understand why the pipeline passed before and passed now. Does the error occur or does it not?

**Comment by dasconnor** [comment]
Also, Fabio's PR is to `fujiDeploy`, can we set this to that branch so that we can integrate your changes?

File: tests/e2e/subnet/suite.go (hunk 1 above)
Comment (felipemadero) [reply tests/e2e/subnet/suite.go:71]: I changed ParseRPCsFromOutput following a previous PR comment, to return error if subnet was already deployed

**Review by dasconnor — APPROVED** [review]
LGTM

**Pull request merged**